Only in orig: bits_container.patch
diff -c orig/ietf-lisp-address-types.yang updated/ietf-lisp-address-types.yang
*** orig/ietf-lisp-address-types.yang	2016-05-23 23:21:50.707847200 +0530
--- updated/ietf-lisp-address-types.yang	2016-05-23 23:15:30.015862257 +0530
***************
*** 449,455 ****
          reference
            "http://tools.ietf.org/html/draft-ietf-lisp-lcaf-10
             #section-4.5";
!         leaf latitude {
            type bits {
              bit N {
                description
--- 449,455 ----
          reference
            "http://tools.ietf.org/html/draft-ietf-lisp-lcaf-10
             #section-4.5";
!         /*leaf latitude {
            type bits {
              bit N {
                description
***************
*** 458,464 ****
            }
            description
              "Bit that selects between North and South latitude.";
!         }
          leaf latitude-degrees {
            type uint8 {
              range "0 .. 90";
--- 458,464 ----
            }
            description
              "Bit that selects between North and South latitude.";
!         }*/
          leaf latitude-degrees {
            type uint8 {
              range "0 .. 90";
***************
*** 480,486 ****
            description
              "Seconds of latitude.";
          }
!         leaf longitude {
            type bits {
              bit E {
                description
--- 480,486 ----
            description
              "Seconds of latitude.";
          }
!         /*leaf longitude {
            type bits {
              bit E {
                description
***************
*** 489,495 ****
            }
            description
              "Bit that selects between East and West longitude.";
!         }
          leaf longitude-degrees {
            type uint16 {
              range "0 .. 180";
--- 489,495 ----
            }
            description
              "Bit that selects between East and West longitude.";
!         }*/
          leaf longitude-degrees {
            type uint16 {
              range "0 .. 180";
***************
*** 588,594 ****
              description
                "AFI address.";
            }
!           leaf lrs-bits {
              type bits{
                bit lookup {
                  description
--- 588,594 ----
              description
                "AFI address.";
            }
!           /*leaf lrs-bits {
              type bits{
                bit lookup {
                  description
***************
*** 605,611 ****
              }
              description
                "Flag bits per hop.";
!           }
          }
        }
        container source-dest-key {
--- 605,611 ----
              }
              description
                "Flag bits per hop.";
!           }*/
          }
        }
        container source-dest-key {
diff -c orig/odl-lisp-proto.yang updated/odl-lisp-proto.yang
*** orig/odl-lisp-proto.yang	2016-05-23 23:21:50.707847200 +0530
--- updated/odl-lisp-proto.yang	2016-05-23 23:15:45.163861655 +0530
***************
*** 493,496 ****
--- 493,500 ----
      notification mappingKeepAlive {
          uses map-register-cache-metadata-container;
      }
+ 
+     container json-container-wrapper{
+         uses mapping-record-container;
+     }
  }
